### PR TITLE
Add config theme.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️

### CODE REVIEW

File: config/params.php [hunk 1]
```diff
@@ -77,6 +77,11 @@
             'url' => UrlGeneratorInterface::class,
             'urlMatcher' => UrlMatcherInterface::class,
         ],
+        'theme' => [
+            'pathMap' => [],
+            'basePath' => '',
+            'baseUrl' => null,
```
Comment (xepozz) [inline config/params.php:83]: Why not empty string?

File: src/Provider/ThemeProvider.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+use Yiisoft\Di\Support\ServiceProvider;
+use Yiisoft\View\Theme;
+
+final class ThemeProvider extends ServiceProvider
+{
+    private array $pathMap;
+    private ?string $basePath;
+    private ?string $baseUrl;
+
+    public function __construct(array $pathMap = [], ?string $basePath = null, ?string $baseUrl = null)
```
Comment (xepozz) [inline src/Provider/ThemeProvider.php:18]:
```suggestion
    public function __construct(array $pathMap, string $basePath, string $baseUrl)
```

File: config/params.php (hunk 1 above)
Comment (terabytesoftw) [reply config/params.php]: done.

File: src/Provider/ThemeProvider.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Provider/ThemeProvider.php]: done.

File: README.md [hunk 3]
```diff
@@ -160,6 +160,13 @@ See ["Logging"](https://github.com/yiisoft/docs/blob/master/guide/en/runtime/log
         'applicationParameters' => 'App\ApplicationParameters',
         'assetManager' => 'Yiisoft\Assets\AssetManager',
     ],
+    'theme' => [
+        // Apply pathMap example: ['@resources/layout' => '@resources/theme'] in yiisoft/app
+        // Apply pathMap example: ['@resources/layout' => '@modulealiases/theme'] in module
+        'pathMap' => [],
+        'basePath' => '',
+        'baseUrl' => null,
```
Comment (rugabarbo) [inline README.md:168]: `'baseUrl' => '',`

Comment (terabytesoftw) [reply README.md:168]: done.

**Comment by samdark** [comment]
:+1

**Pull request merged**